src/webapp/pages/blank-template/BlankTemplatePage.tsx

```diff
@@ -0,0 +1,17 @@
+import i18n from "../../../locales";
+import { DialogContentText } from "@material-ui/core";
+import React from "react";
+
+export default function BlankTemplatePage(): React.ReactElement {
+    return (
+        <React.Fragment>
+            <h3>{i18n.t("User with no permissions")} </h3>
+            <DialogContentText>
+                {i18n.t(
+                    "It seems this user account does not have any permission." +
+                        " Please get in touch with the admin if you need access to these sections."
+                )}
```

Comment:
This one is tricky. We cannot do `i18n.t("abc" + "def")`. In en.pot, we'll see these two strings are not present. That's because the gettext extraction is pretty crude: it simply looks for things like `i18.t("Message")`. You can use a interpolated string (backtick), which allows long lines / multiline, or `i18n.t("abc") + i18n.t("def")`.